Conan.VisualStudio/Services/SolutionEventsHandler.cs

```diff
@@ -1,14 +1,6 @@
-using Conan.VisualStudio.Core;
```

Comment:
for me whole file appears to be useless - contains just stubs, is it actually needed for something?

Reply:
I'll have a look at it, not sure now. This PRs are just little removals, noone tries to remove it all